components/ai_chat/resources/page/state/ai_chat_context.tsx

```diff
@@ -37,12 +41,19 @@ const defaultContext: AIChatContext = {
   userRefreshPremiumSession: () => { },
 
   editingConversationId: null,
-  setEditingConversationId: () => { }
+  setEditingConversationId: () => { },
+
+  showSidebar: false,
+  toggleSidebar: () => { }
 }
 
 export const AIChatReactContext =
   React.createContext<AIChatContext>(defaultContext)
 
+export function useIsSmall() {
+  return useMediaQuery('(max-width: 1024px)')
```

Comment:
Is it worth just putting this as a context prop so it doesn't have to evaluate and listen from every consuming component?

Reply:
tbh, I think the performance impact of this is probably pretty minimal - if we were concerned we could set up a cache in `useMediaQuery` but I don't think its worth it at the moment.